Common/src/CConfig.cpp

```diff
@@ -2887,6 +2884,8 @@ void CConfig::SetConfig_Parsing(istream& config_buffer){
             newString.append("WRT_SOL_FREQ is deprecated. Use OUTPUT_WRT_FREQ instead.\n\n");
           if (!option_name.compare("WRT_SOL_FREQ_DUALTIME"))
             newString.append("WRT_SOL_FREQ_DUALTIME is deprecated. Use OUTPUT_WRT_FREQ instead.\n\n");
+          if (!option_name.compare("CONV_CRITERIA"))
+            newString.append("CONV_CRITERIA is deprecated.\n\n");
```

Comment:
Dammmm really?! I guess if something is a residual it uses the residual criteria (criterion?) and if it is a coefficient it uses Cauchy?
Mind throwing that in the message? This is another one that will break every config...

Reply:
Exactly. In COutput::Convergence_Monitoring there is a distinction for `CONV_FIELD` between `HistoryFieldType::COEFFICIENT` and `HistoryFieldType::RESIDUAL`.

I'll add that information to the error msg 👍

Reply:
actually... do you think we should give folks a pass and just print a warning (with that exact message) and say that we will deprecate in the future? When we go to 7.2.0 (which could happen soon if we merge the hybrid parallel AD and the Scalar solvers)

Reply:
Sure, that also requires readding the option. I'll try to remember to enforce that for 7.2.0

Reply:
Ok I made it a warning but as it is thrown at config parsing in Postprocessing I guess not lot of people will see it anyway. 

I guess one can `git revert aa7393f` at some point in the future.

And CONV_CRITERIA is a stringOption so one can type in there whatever one likes :) no optionMap required